From 59c58e8a090f3335c02d87c3d13ae707ddc1557f Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 08 Dec 2022 12:12:25 +0200
Subject: [PATCH] [SW-66327] pif_arc: gaudi3: fix ltssm register

copy/paste error caused the wrong register to be read

Change-Id: I898b52d36712095b65032eea215b7cdad809f0be
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 22f5c4a..3ab8b92 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -972,7 +972,7 @@
 		unsigned int prstn, ltssm, smlh, rdlh;
 
 		prstn = regmap_readl(mmD0_PSOC_GLOBAL_CONF_PRSTN);
-		ltssm = regmap_readl(mmD0_PSOC_GLOBAL_CONF_PRSTN);
+		ltssm = regmap_readl(mmD0_PCIE_AUX_LTSSM_EN);
 		smlh = regmap_readl(mmD0_PCIE_AUX_SMLH_LINK_UP);
 		rdlh = regmap_readl(mmD0_PCIE_AUX_RDLH_LINK_UP);
 
